common/protob/messages-management.proto

```diff
@@ -140,12 +141,20 @@ message ApplySettings {
     optional string label = 2;
     optional bool use_passphrase = 3;
     optional bytes homescreen = 4;
-//    optional PassphraseSourceType passphrase_source = 5;  DEPRECATED
+    optional PassphraseSourceType _passphrase_source = 5 [deprecated=true];  
```

Comment:
would it work to declare this as `uint32` so that we can avoid the enum?

Reply:
that should do imho.